preamble.tex

```diff
@@ -1,41 +1,27 @@
 \section*{Preamble}
-The establishment of Beta Theta Pi at Worcester Polytechnic Institute was carried out by the Founding Fathers listed below who on April 14, 2014 thereby adopted the Constitution and Bylaws, which embodied the Code of Regulation and the purpose of the Eta Tau chapter of Beta Theta Pi.
+\label{sec:preamble}
 
-\begin{multicols}{3}
-        \begin{itemize}[leftmargin=0pt]
-\item[] Diego Martin Adrienzen
-\item[] Nicholas J. Aleles
-\item[] Rohit Bansal
-\item[] Aaron Benjamin
-\item[] Joseph M. Brown
-\item[] Tanner B. Buxton
-\item[] Enrique Delacruz
-\item[] Serhan Delareyna
-\item[] Michael Kurt Delia
-\item[] David M. DiStefano
-\item[] Connor Ellison
-\item[] Eric Faust
-\item[] Micah Flock
-\item[] Sam Friedman
-\item[] Mohammad A. Ghani
-\item[] Marc Green
-\item[] Kirk Grimsley
-\item[] Eric Guleksen
-\item[] Brian Hamilton
-\item[] Justin Peter Hess
-\item[] Tyler T. Howard
-\item[] Andrew W. Iovanna
-\item[] Michael Jameson
-\item[] David Karbassi
-\item[] Spencer Keilich
-\item[] Jeremy Kibby
-\item[] Jonathan Labrie
-\item[] Tushar Narayan
-\item[] Scott D. Osborn
-\item[] Randy Perez
-\item[] Grant Raymond
-\item[] Shreyas Renganathan
-\end{itemize}
+The establishment of Beta Theta Pi at Worcester Polytechnic Institute was
+carried out by the Founding Fathers listed below who on April 14, 2014 thereby
+adopted the Constitution and Bylaws, which embodied the Code of Regulation and
+the purpose of the Eta Tau chapter of Beta Theta Pi.
 
-\end{multicols}
+\begin{tabular}[h]{l l l}
```

Comment:
To me this looks weird because it butts the table right up against the previous paragraph

Reply:
Added more space after the paragraph